### DISCUSSION TITLE
mbr.hex for nrf5340 proting needed

### DISCUSSION THREAD

**Discussion by rkudaj** [adafruit/Adafruit_nRF52_Bootloader#298] [Q&A]
Hi,
I'm trying to port this bootloader project onto nrf5340 micro, currently I m searching for mbr.hex file just for nrf5340.
For nrf52xx this file is located lib/softdevice/mbr/hex folder. File is named "mbr_nrf52_2.4.1_mbr.hex". 
Does anyone know where this file was taken from? where I can find source code to compile for nrf5340 ?

regards

**Comment by dmalnati** [comment]
Hi,

I don't know if what I did is exactly what you're looking for, but I was able to get this bootloader working for the nRF52840.  I compiled and flashed the bootloader directly using a J-LINK.

Here is a DevZone post where I detailed my steps: ([link](https://devzone.nordicsemi.com/f/nordic-q-a/97314/uf2-image-and-bootloader)).

Here is a post on this github discussion about some issues along the way (eg flash partitions) ([link](https://github.com/adafruit/Adafruit_nRF52_Bootloader/discussions/295)).

I have not used my approach extensively yet and can't vouch for 100% operation, but seems to work so far.

**Comment by rkudaj** [comment]
hi dmalnati,

thanks for help, however I m looking for official mbr file (sources) for nrf5340,

regards,
